src/Mortgage/Mortgage.spec.js

```diff
@@ -4,32 +4,44 @@ import { txUtils } from 'decentraland-eth'
 import { db } from '../database'
 import { Mortgage } from '../Mortgage'
 import { Parcel } from '../Parcel'
+import { ASSET_TYPE } from '../Asset'
+import { Publication } from '../Publication'
 
 describe('Mortgage', function() {
+  const expires_at = new Date().getTime() * 1000
+  const is_due_at = 15000
+  const payable_at = 1000
+  let mortgage = {}
```

Comment:
I did what you said on the [`beforeEach` method](https://github.com/decentraland/marketplace/pull/235/files/d3611929714aed9e0e2425c641bfcc4da81903b0#diff-b472fc2ffbb046805ee9a677076787ebR16)

Reply:
Yes but what I meant is that we don't need to use a `beforeEach` that redefines the object mortgage each time if we use on each test something like

Reply:
Oh, I can't still see the advantage of not doing it in a test. Is it ok for you to keep the way it is?

Reply:
The advantages are:
- Variables can still be called `mortgage` instead of `mortgage2` (code readability)
- We just build mortgages on tests that actually need it, and not all of them with a `beforeEach`
- Tests become more encapsulated since they wouldn't use an external estate (the `let mortgage = {}` variable)